src/scripts/install.sh

```diff
@@ -12,81 +12,20 @@ fi
 Install_AWS_CLI() {
     echo "Installing AWS CLI v2"
     cd /tmp || exit
-    # Platform check
-    if uname -a | grep "Darwin"; then
-        export SYS_ENV_PLATFORM=macos
-    elif uname -a | grep "x86_64 GNU/Linux"; then
-        export SYS_ENV_PLATFORM=linux_x86
-    elif uname -a | grep "aarch64 GNU/Linux"; then
-        export SYS_ENV_PLATFORM=linux_arm
-    elif uname -a | grep "x86_64 Msys"; then
-        export SYS_ENV_PLATFORM=windows
-    elif grep "Alpine" /etc/issue >/dev/null 2>&1; then
-        export SYS_ENV_PLATFORM=linux_alpine
-    else
-        echo "This platform appears to be unsupported."
-        uname -a
-        exit 1
-    fi
+
+    eval "$SCRIPT_UTILS"
+    detect_os
```

Comment:
These two lines should be outside the function. They have to be the first thing to happen because they will provide you with information about the operating system.

You should not have to run any sort of command to figure out what OS you are on. This will be done by `detect_os`. All you have to do is access `SYS_ENV_PLATFORM`.